### ISSUE TITLE
Allow Multi Level for writer Multi tag feature

### ISSUE THREAD

**Issue by Git5000** [tinylog-org/tinylog#174]
Tinylog 2.2 supports writers with more than one tag like 
`writer.tag = tag1, tag2, tag3.`
It would be useful if the tags could be set to different log levels. Currently all tags use the same level.

This would allow e.g.
`writer.level = INFO, WARN, TRACE`

It should only be allowed to either set one level (which then applies to all tags) or exactly the same amount of levels than tags. Otherwise an exception or error should be risen. All other case lead to confusion.

Implementation should be easy as the writer matrix needs to be filled just more sparsely and corresponding to the level.

**Comment by pmwmedia** [comment]
I like the idea of applying different severity levels for different tags. I just would adjust the configuration syntax:

```
writer.level = INFO, WARN@tag1, TRACE@tag2
```

The advantages are that users don't need to set a severity level for each tag (default is the untagged severity level), it is clearer which severity level belongs to which tag, and the current configuration syntax would still work:

```
writer.tag   = tag1, tag2, tag3
writer.level = INFO
```

**Comment by Git5000** [comment]
Yes that is a good idea. I was a bit worried with my idea that the two lists have to have a 1:1 correspondence. This is solved with your proposal. The only disadvantage is that is more text. One probably would like somehow to group them. But we should also not make the parsing of the string to complicated and causes bugs. That is never a good idea.

So one could do something like this - but I actually would advise against my own suggestion below ;-)
`
writer.level = INFO, WARN@(tag1,tag,tag3), TRACE@(tag4,tag5)
`
so we better try yours.

**Comment by Git5000** [comment]
Another idea came to mind. What about changing the tag not the level entry like:

`writer.tag= tag1(INFO), tag2(ERROR), tag3(DEBUG), tag4(INFO)`
or 
`writer.tag= tag1@INFO, tag2@ERROR, tag3@DEBUG, tag4@INFO`
or
`writer.tag=INFO@tag1, ERROR@tag2, DEBUG@tag3, INFO@tag4`

because then you have the information at one place where you probably most think about it.

For compatibility of course the old syntax needs to remain, i.e.
`writer.level = INFO `
sets a default level for all tags without explicit level. But the level tag syntax would not changed.

PS: You can assign it to me if we go ahead with it.

**Comment by pmwmedia** [comment]
I like your idea `writer.tag = tag1@INFO, tag2@ERROR, tag3@DEBUG, tag4@INFO` 👍

**Comment by pmwmedia** [comment]
Fixed by #177
